src/static/json/RDD/RDD.json

```diff
@@ -0,0 +1,93 @@
+{
+  "visibleName": "Radial dose distribution",
+  "functionName": "AT_D_RDD_Gy",
+  "description": "TODO",
+  "xTitle": "Distance [m]",
+  "yTitle": "Dose [Gy]",
+  "plot": true,
+  "unit": null,
+  "formItems": [
+    {
+      "type": "entry_module",
+      "parameterName": "r_m",
+      "startholder": "0.0000000001",
+      "endholder": "0.001",
+      "intervalType": "points",
+      "validations": {
+        "type": "float",
+        "min": "0.000000000000000.1",
+        "max": "1000.0"
+      }
+    },
+    {
+      "type": "input",
+      "parameterName": "E_MeV_u",
+      "label": "Energy [MeV]",
+      "placeholder": "150",
+      "defaultValue": 150,
+      "validations": {
+        "type": "float",
+        "min": "0.000001",
+        "max": "1000000.0"
+      }
+    },
+    {
+      "type": "select",
+      "parameterName": "particle_no",
+      "label": "Particle",
+      "values": "particles",
+      "asManyAsPoints": false,
+      "defaultValue": 1001,
+      "description": "Choose particle type"
+    },
+    {
+      "type": "select",
+      "parameterName": "material_no",
+      "label": "Material",
+      "values": "materials",
+      "defaultValue": 1,
+      "description": "Choose material type"
+    },
+    {
+      "type": "select",
+      "parameterName": "rdd_model",
+      "label": "RDD model",
+      "values": "rdd_models",
+      "asManyAsPoints": false,
+      "defaultValue": 2,
+      "description": "Choose RDD model"
+    },
+    {
+      "type": "select",
```

Comment:
@marwin1991 @adudzik rdd_parameters store the array of parameters. It is fixed size array of 3 elements. Is it possible to use such data types here ?

Reply:
First it needs to add dictionary. Then use it. Same as materials or particles.
https://libamtrackweb.readthedocs.io/en/latest/jsons.html#gcjson

Reply:
But I'd like to provide the values (up to 3 numbers) in the input text fields in the web interface. If not possible I'll add another C wrapper to reduce the number of RDD parameters to a single one.

Reply:
I'm not sure if I understood you correctly but it's not possible (in currently supported input types) to insert more than one value in single input.  It's not necessary to create another wrapper, we can extend our form items with e.g. multiselect or something like that.

Reply:
RDD models needs extra parameters which are put together into single array. Some models (i.e. Geiss) require a single number which goes into rdd_parameter[0], rest is ignored. Other models require 2 or 3 numbers.